custom/opm/filters.py

```diff
@@ -51,6 +70,56 @@ def get_hierarchy():
     return hierarchy
 
 
+def user_data_as_hierarchy():
```

Comment:
Initially I was debating between modifying `get_hierarchy` and adding this new one. But since `get_hierarchy` touches all the reports, I have created another one.